readme.md

```diff
@@ -34,6 +34,7 @@ A curated list of awesome things related to <a href='https://vast.io/'>VAST</a>
 
 ## Community Projects
 
+- [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence/tree/main/users/) -  A Nix-based DevOps framework for defining declarative and reproducible environments and jobs. 
```

Comment:
```suggestion
- [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence) -  A Nix-based DevOps framework for defining declarative and reproducible environments and jobs. 
```

Reply:
Sorry about that. The `data-science-threat-intelligence` is another story.